packages/guides/src/ReferenceResolvers/InterlinkReferenceResolver.php

```diff
@@ -29,7 +32,12 @@ public function resolve(LinkInlineNode $node, RenderContext $renderContext, Mess
 
         $domain = $node->getInterlinkDomain();
         $target = $node->getTargetReference();
-        if (!$this->inventoryRepository->hasInventory($domain)) {
+        $group = $node instanceof DocReferenceNode ? 'std:doc' : 'std:label';
+
+        try {
+            $inventory = $this->inventoryRepository->getInventory($domain);
+            $link = $this->inventoryRepository->getLink($domain, $group, $target);
```

Comment:
Why are this separate methods? Ther inventory contains the links doesn't it?

Reply:
we need the inventory to get the base url